### PR TITLE
setup header

### PR DESCRIPTION
**Pull request by darshanasen**
I have implemented a header and added some content. Could use some feedback on the chosen structure.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,6 +7,12 @@
   <title>Git Collab Lesson!</title>
 </head>
 <body>
-  
+  <header>
+    <p>This is the title of my cool page! 🤘🏽</p>
```
Comment (susan8098) [inline index.html]: This seems like a title for the page, maybe it should be a heading tag like `<h1>` instead of a `<p>` tag?

File: index.html [hunk 2]
```diff
@@ -7,6 +7,12 @@
...
+
+    <div class="navigation">
+      <a href="#">Home</a>
```
Comment (susan8098) [inline index.html:14]: This is great!

**Review by susan8098 — CHANGES_REQUESTED** [review]
Just a few things I spotted that needs review/change

File: index.html (hunk 1 above)
Comment (darshanasen) [reply index.html]: Excellent feedback! Thanks, Susan!

**Comment by darshanasen** [comment]
Made some changes! Let me know if these are better semantic choices.

**Review by susan8098 — APPROVED** [review]
This looks great and amazing, nice catch on the changing of `<nav>` 
GOOD TO GOOOOOO 🚀🚀🚀🚀

**Pull request merged**